scripts/normalize-by-median.py

```diff
@@ -189,19 +188,19 @@ def normalize_by_median_and_check(input_filename, htable, single_output_file,
 
     total_acc = None
     discarded_acc = None
-
+    corrupt = None
     if single_output_file:
         if single_output_file is sys.stdout:
             output_name = '/dev/stdout'
         else:
             output_name = single_output_file.name
         outfp = single_output_file
-
     else:
         output_name = os.path.basename(input_filename) + '.keep'
         outfp = open(output_name, 'w')
 
-    with CatchIOErrors(input_filename, outfp, fail_save, htable, force, norm):
+    with CatchIOErrors(input_filename, outfp, fail_save, htable, force) as \
```

Comment:
note: this still needs to be addressed--don't think there's a test in place for handle_error on single_output_file